pom.xml

```diff
@@ -872,6 +872,36 @@
         <java.level>8</java.level>
       </properties>
     </profile>
+    <profile>
+      <id>deprecated-java.level-7</id>
+      <activation>
+        <property>
+          <name>java.level</name>
+          <value>7</value>
```

Comment:
There may be some plugins using 6 out there still, though I suppose they are not using a modern POM either…

Reply:
There is no good way to do conditional profiles or to chain them. Hit the same in Java 11 support PRs. So I just did the check for Java 7, but can copy-paste if needed

Reply:
I think it is good enough, just noting this.